napari/layers/image/image.py

```diff
@@ -337,6 +350,64 @@ def rendering(self, rendering):
         self._rendering = rendering
         self.events.rendering()
 
+    @property
+    def complex_func(self):
```

Comment:
I don't love the name `complex_func`, is there something else that can be more explicit, maybe `complex_converter`? Not sure I love that either.

Reply:
renamed to `complex_rendering`, which sticks with other terminology.